packages/@sanity/core/src/actions/graphql/listApisAction.js

```diff
@@ -17,12 +17,17 @@ module.exports = async function listApisAction(args, context) {
   }
 
   if (endpoints && endpoints.length > 0) {
+    const {projectId} = client.config()
     output.print('Here are the GraphQL endpoints deployed for this project:')
     endpoints.forEach((endpoint, index) => {
-      output.print(`${index + 1}.  ${chalk.bold('Dataset:')}     ${endpoint.dataset}`)
-      output.print(`    ${chalk.bold('Tag:')}         ${endpoint.tag}`)
+      const {dataset, tag} = endpoint
+      const url = `https://${projectId}.api.sanity.io/v1/graphql/${dataset}/${tag}`
```

Comment:
Would be nicer to use some centralized endpoint url resolver here. Does it exist?